draft-ietf-ccwg-rfc5033bis.md

```diff
@@ -332,8 +334,8 @@ asymptotic behavior.
 
 In contexts where differing congestion control
 algorithms are used, it is important to understand whether
-an alternate congestion control algorithm can induce more
-harm to sharing flows than existing
+a proposal can induce more
+harm to sharing flows than for the existing
 defined methods. The measure of harm is not restricted to the equality
```

Comment:
This does not read well: "In contexts where differing congestion control
algorithms are used, it is important to understand whether
a proposal can induce more
harm to sharing flows than for the existing
defined methods." 

Proposal: "In contexts where differing congestion control
algorithms are used, it is important to understand whether
a proposal can induce more
harm than previously defined algorithms to flows sharing a common bottleneck."

Reply:
better as far as I can see
but ... Looking at this, i worry about whether "can" could be misread?
 /can/could result in/ ?

Reply:
You are right, but I think this may be OBE. I was commenting on an old PR. The whole text has been reformatted already.

Reply:
I will open a specific PR.